docs/apps/realtime-log-analytics.md

```diff
@@ -43,23 +37,22 @@ fastlyGetUniqueVisitorsByCountry
 3. Create the following collections in your GDN account:
 
 ```js
-fastly_users (global)
-fastly_logs (global)
-fastly_http_url_stats_1m (global)
-fastly_http_response_code_stats_1m (global)
-fastly_http_response_latency_stats_1m (global)
-fastly_http_error_response_code_stats_1m (global)
-fasty_unique_visitor_by_country_stats_1m (global)
+users (global)
+logs (global)
+http_url_stats_1m (global)
+http_response_code_stats_1m (global)
+http_response_latency_stats_1m (global)
+http_error_response_code_stats_1m (global)
+unique_visitor_by_country_stats_1m (global)
 ```
 
 4. On the development machine, run the following commands in a console:
 
 ```
-1. git clone git@github.com:Macrometacorp/demo-fastly-jsc8-realtime-logs.git
-2. cd fastly-edgeworker-log-analytics
-3. git fetch
-4. npm install
-5. npm run start
+1. git clone git@github.com:Macrometacorp/demo-jsc8-realtime-logs.git
```

Comment:
What are these numbers doing here? Won't they cause an error if I copy and paste the code block in the terminal?